package.json

```diff
@@ -16,28 +16,26 @@
     "extension",
     "sign"
   ],
-  "main": "dist/connector.js",
-  "dependencies": {
-    "babel-runtime": "^6.26.0"
-  },
+  "main": "connector.js",
   "peerDependencies": {
     "aws-sdk": "^2.83.0",
     "elasticsearch": "^13.2.0"
   },
   "devDependencies": {
     "aws-sdk": "^2.83.0",
-    "babel-cli": "^6.24.1",
-    "babel-eslint": "^7.2.3",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.0",
+    "chai": "^4.1.2",
     "elasticsearch": "^13.2.0",
     "eslint": "^4.2.0",
-    "minimist": "^1.2.0"
+    "minimist": "^1.2.0",
+    "mocha": "^3.5.3",
+    "nyc": "^11.2.1",
+    "sinon": "^4.0.0"
   },
   "scripts": {
-    "prepublish": "npm run build",
-    "build": "rm -rf dist; mkdir dist; babel ./connector.js -o dist/connector.js",
-    "test": "node test/local"
+    "test": "nyc --all mocha",
+    "integration-test": "mocha test/integration",
```

Comment:
nothing changes here with how you ran it before, just now you do `npm run integration-test -- [env, etc]` and it reports the pass/fail as a mocha report.